qemu/tests/fio_perf.py

```diff
@@ -95,38 +112,52 @@ def run(test, params, env):
     :param env: Dictionary with test environment
     """
 
-    def fio_thread(session, run_cmd, cmd_timeout):
+    def fio_thread():
```

Comment:
I'm not sure if remove these params would lead a problem, it is better to keep them for making the code readable.